hikari/impl/special_endpoints.py

```diff
@@ -1785,15 +1841,22 @@ def value(self) -> undefined.UndefinedOr[str]:
         return self._value
 
     @property
-    def required(self) -> undefined.UndefinedOr[bool]:
+    def required(self) -> bool:
```

Comment:
```suggestion
    @deprecation.deprecated
    def required(self) -> bool:
```

Reply:
deprecation.deprecated won't make it to the next release without including `typing_extensions` as a dependency since MyPy's next release is planned for the 4th of April https://github.com/python/mypy/issues/14916

Reply:
Ah, thats my bad, forgot we already had it in master